app/javascript/packages/document-capture/components/document-capture-warning.tsx

```diff
@@ -100,13 +100,15 @@ function DocumentCaptureWarning({
           />
         </div>
 
-        {!isFailedDocType && remainingAttempts <= DISPLAY_ATTEMPTS && (
-          <p>
-            <HtmlTextWithStrongNoWrap
-              text={t('idv.failure.attempts_html', { count: remainingAttempts })}
-            />
-          </p>
-        )}
+        {!isFailedDocType &&
+          !isFailedSelfieLivenessOrQuality &&
+          remainingAttempts <= DISPLAY_ATTEMPTS && (
+            <p>
+              <HtmlTextWithStrongNoWrap
+                text={t('idv.failure.attempts_html', { count: remainingAttempts })}
```

Comment:
I tested this out in terms of the testing plan and it worked! I almost approved it but I did have two concerns:

- CI is failing (seems to be due to a missing changelog though)
- I wanted to test this out locally since you said you had a hard time replicating it locally. Looking in the translation files, it does seem like this text is used in 3 different places (one of which adds the `...to add your ID online` bit).
    - I had a thought that since one of the error texts was in the "phone" section, maybe the double text is only seen on mobile? But I ran into some errors locally (details below*). Before I went too far into the weeds trying to fix my issues before approving, I thought I'd ask if you tried this on mobile and still couldn't get it to show. If so then I'm okay approving as is.

*The errors I ran into were that couldn't choose the upload file button, and couldn't close out of the selfie SDK. I ran into these issues on `main` too so I don't think it's a "this branch" problem, and probably a "my local setup" problem.

Reply:
I did test with mobile, yes. I also tried this on the `dev` site. I just retested with mobile on dev and I'm still not having any luck reproducing it.

But! The test that I added does seem to reproduce the problem, so I'm pretty confident in this fix.

Reply:
Added a changelog.

Reply:
> I had a thought that since one of the error texts was in the "phone" section,
Can you point me to this? Curious to look at the phone specific part.

Reply:
Sure! It looks like we have this text as `idv/phone/warning/attempts_html` [here](https://github.com/18F/identity-idp/blob/main/config/locales/idv/en.yml#L116).

Then we also have it as `idv/failure/attempts_html` [here](https://github.com/18F/identity-idp/blob/main/config/locales/idv/en.yml#L63).

And finally as `idv/warning/attempts_html` [here](https://github.com/18F/identity-idp/blob/main/config/locales/idv/en.yml#L325).

I didn't dig too far into its usage for all 3 though.

Reply:
Thankfully, the [here](https://github.com/18F/identity-idp/blob/main/config/locales/idv/en.yml#L116) (`idv/phone/warning/attempts_html`) isn't used in this part of the codebase as far as I can tell. Thanks for highlighting these!